src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -175,9 +191,12 @@ private void WriteRunLogToErrorList(RunLog runLog)
                         SarifError sarifError = new SarifError(document)
                         {
                             Region = region,
-                            ErrorCode = result.RuleId,
+                            RuleId = result.RuleId,
+                            RuleName = rule?.Name,
                             Kind = ResultKind.Note,
-                            Message = message,
+                            Category = "Related Location", // or should we prefer original result category?
```

Comment:
I think it's right the way it is. If you ask the question "how many Design issues do I have", you don't want the answer inflated by the related locations.